spin/apps/03-spin-ai/magic-8-ball-rust/spin.toml

```diff
@@ -15,3 +15,10 @@ route = "/magic-8"
 [component.build]
 command = "cargo build --target wasm32-wasi --release"
 watch = ["src/**/*.rs", "Cargo.toml"]
+
+[[component]]
+source = { url = "https://github.com/fermyon/spin-fileserver/releases/download/v0.0.3/spin_static_fs.wasm", digest = "sha256:38bf971900228222f7f6b2ccee5051f399adca58d71692cdfdea98997965fd0d" }
```

Comment:
by step 3, the front end has not entered the picture. but i can see how this would be helpful for demo purposes. Should we swap steps 3 and 4?

Reply:
That's a good idea. However a large change. For now, I'm just removing the fileserver from the rust app in step 3, as having it there was an oversight.